apps/hash-ai-worker-py/app/__main__.py

```diff
@@ -7,8 +7,8 @@
 from temporalio.client import Client
 from temporalio.worker import Worker
 
-from app.activities import complete
-from app.workflows import DemoWorkflowPy
+from .activities import complete
+from .workflows import DataTypeWorkflow, EntityTypeWorkflow, PropertyTypeWorkflow
```

Comment:
Can I ask why you've moved to relative imports?

Reply:
This must be due to some autoimports. I don't know if this is configurable but if it's not I'd prefer to keep it relative as otherwise it will become inconsistent. I'll do some research

Reply:
Seems to depend where VSCode thinks the root is. Made it consistent in [`cc45602` (#2699)](https://github.com/hashintel/hash/pull/2699/commits/cc4560250baa6258c56a2fddb8b5b14f6c736c6b)